packages/devextreme-angular-generator/src/metadata-generator.ts

```diff
@@ -226,7 +230,18 @@ export default class DXComponentMetadataGenerator {
                     return result;
                 }, []);
 
-            const widgetMetadata: WidgetComponent & File = {
+
+            const hasExplicitTypes = !!widget.OptionsTypeParams?.length;
+
+            const filterReexports = (reexports?: string[]) : string[] => (
+                reexports
+                    ? reexports.filter(
+                        (item) => item !== 'default' && (hasExplicitTypes && item !== 'ExplicitTypes'),
```

Comment:
Can the condition be simplified?
```suggestion
                        (item) => item !== 'default' && item !== 'ExplicitTypes',
```

Reply:
fixed

Reply:
fixed